src/common/layouts/Header/Header.jsx

```diff
@@ -43,11 +46,16 @@ const Header = props => (
       <Col lg={6} xl={5}>
         <Row type="flex" justify="end">
           <Col className="nav-item-container">
-            <DropdownMenu
+            {props.shouldDisplayHoldingPen && <DropdownMenu
               title="Tools"
               titleClassName="nav-item"
               items={TOOL_LINKS}
-            />
+            />}
+            {!props.shouldDisplayHoldingPen && <DropdownMenu
+              title="Tools"
+              titleClassName="nav-item"
+              items={TOOL_LINKS_BETAUSER}
```

Comment:
instead of duplicating the `Component` in `render`, you can create a function `getToolLinksForUser` and implement the logic there set only `items={getToolLinksForUser()}`